src/community/resilience.jl

```diff
@@ -0,0 +1,111 @@
+#=
+Some functions from *Universal resilience patterns in complex networks*
+by Gao et al. (2016)
+=#
+
+using StatsBase: mean, std
+
+"""
+    s_in(N::AbstractUnipartiteNetwork)
+
+Computes the vector of incoming weighted degrees of an unipartite network.
+"""
+s_in(N::AbstractUnipartiteNetwork) = sum(N.A, dims=2)
+
+"""
+    s_out(N::AbstractUnipartiteNetwork)
+
+Computes the vector of outgoing weighted degrees of an unipartite network.
+"""
+s_out(N::AbstractUnipartiteNetwork) = sum(N.A, dims=1)'
+
+"""
+    s_mean(N::AbstractUnipartiteNetwork)
```

Comment:
Is this working for binary networks _and_ quantitative networks?

Reply:
I both have unit tests. `sum` does not always work for both types, that's why I use `sum(N.A,...)` to be certain.

Unit tests use:
```
UnipartiteQuantitativeNetwork
UnipartiteNetwork
UnipartiteProbabilisticNetwork
```
using `symmetry` and `heterogeneity` so all functions run during testing.